examples/add_images.py

```diff
@@ -0,0 +1,24 @@
+"""
+Display two image layers using the add_image API
+"""
+
+import sys
+from PyQt5.QtWidgets import QApplication
+
+from skimage import data
+from skimage.color import rgb2gray
+from napari_gui import Window, Viewer
+
+
+# starting
+application = QApplication(sys.argv)
+
+# create the viewer and window
+viewer = Viewer()
+window = Window(viewer)
+# add the first image
+viewer.add_image(rgb2gray(data.astronaut()), {})
+# add the second image
+viewer.add_image(data.camera(), {})
+
+sys.exit(application.exec_())
```

Comment:
`exec` is no longer a reserved keyword in Python 3 so `exec_` is really just for Python 2 compatibility reasons and since we are Python 3-only we can use the regular version :)

```suggestion
sys.exit(application.exec())
```

Reply:
fixed